utils/btrfs-progs/files/btrfs-scan.init

```diff
@@ -1,9 +1,7 @@
-#!/bin/sh /etc/rc.common
-# Copyright (C) 2014 OpenWrt.org
+#!/bin/sh
 
-START=19
-
-start() {
-	grep -q btrfs /proc/filesystems && /usr/bin/btrfs device scan
+preinit_btrfs_scan() {
+	grep -vq btrfs /proc/filesystems || btrfs device scan
```

Comment:
So this is interesting :). You changed /usr/bin/btrfs to just btrfs. If I take a look at my OpenWrt device, ps returns mostly binaries with the prefix.

I don't know if there's a standard here that should be followed.